test/matchers/log_format_test.go

```diff
@@ -53,14 +53,14 @@ var _ = Describe("Log Format matcher tests", func() {
 	It("match same time value", func() {
 		timestamp := "2013-03-28T14:36:03.243000+00:00"
 		nanoTime, _ := time.Parse(time.RFC3339Nano, timestamp)
-		Expect(types.AllLog{ContainerLog: types.ContainerLog{ViaQCommon: types.ViaQCommon{Timestamp: nanoTime}}}).
-			To(FitLogFormatTemplate(types.AllLog{ContainerLog: types.ContainerLog{ViaQCommon: types.ViaQCommon{Timestamp: nanoTime}}}))
+		Expect(types.AllLog{ContainerLog: types.ContainerLog{ViaQCommon: types.ViaQCommon{TimestampLegacy: nanoTime}}}).
```

Comment:
shouldnt we be testing for population of both values since we are sending both for backwards compatibility?

Reply:
The original idea was we wanted to remove the original field because we determined it has no value.   As an alternative, we kept both to help ensure no regression issues.    Should we be concerned about testing an unused field?